docs/modules/ROOT/pages/web-frameworks.adoc

```diff
@@ -206,6 +206,24 @@ This will configure the build to export the static files:
   }
 ----
 
+[#vite-config]
+=== Vite Configuration
+
+You must update `vite.config.ts` to let Vite know this is a Multiple Page Application so any unknown URL gets a `404` instead of routing to `index.html`.
+.vite.config.ts:
```

Comment:
```suggestion
vite.config.ts:
```
remove .

Reply:
The dot is important for asciidoc code block title